static/styles/compose.css

```diff
@@ -184,11 +184,37 @@
     margin: auto;
 }
 
+#compose_top_right {
+    position: absolute;
+    right: 16px;
+    top: -1px;
+
+    a {
+        color: hsl(0, 0%, 0%);
```

Comment:
This way of setting the color for the `a` tag results in our needing to do something for the night theme as well.  I wonder if we can do `color: inherit` or something?  Or more generally have it share color/opacity CSS with the `x`.

Reply:
Changing it to buttons, so no removed this CSS.